Touchup install and quickstart (#315)
zeek/zeek-docs: get-started/quickstart.rst

     $ zeek -i en0 -C

> Commands which use a network device require root access in order to monitor
> traffic.

Reviewer: This wording confused me a little at first. I switched around the ordering a little, plus root access *may* be required but not always.

```suggestion
Root access is typically required to run commands which monitor a network device.
```
